test/all.dart

```diff
@@ -3,101 +3,22 @@
 // BSD-style license that can be found in the LICENSE file.
 
 import 'package:linter/src/io.dart';
-import 'package:unittest/unittest.dart';
+import 'package:test/test.dart';
 
 import 'config_test.dart' as config_test;
 import 'formatter_test.dart' as formatter_test;
 import 'integration_test.dart' as integration_test;
 import 'io_test.dart' as io_test;
 import 'linter_test.dart' as linter_test;
-import 'mocks.dart';
 import 'project_test.dart' as project_test;
 import 'pub_test.dart' as pub_test;
 
-main() {
-  // Tidy up the unittest output.
-  filterStacks = true;
-  formatStacks = true;
-
-  // useCompactVMConfiguration();
-  //useTimingConfig();
-
-  // Redirect output.
-  outSink = new MockIOSink();
-
-  linter_test.main();
-  pub_test.main();
-  io_test.main();
-  formatter_test.main();
-  config_test.main();
-  project_test.main();
-  integration_test.main();
-}
-
-void useTimingConfig() {
-  unittestConfiguration = new TimingTestConfig();
-}
-
-class TimingTestConfig extends SimpleConfiguration {
```

Comment:
I think the data is there, it just needs to be surfaced. Likely a bug request for pkg/test

Not sure what you mean by `IOSink`...